Tests/NIOPosixTests/SystemCallWrapperHelpers.swift

```diff
@@ -37,7 +37,7 @@ public func measureRunTime(_ body: () throws -> Int) rethrows -> TimeInterval {
 
 public func measureRunTimeAndPrint(desc: String, body: () throws -> Int) rethrows -> Void {
     print("measuring: \(desc)")
-    print("\(try measureRunTime(body))s")
+    print("\(try measureRunTime(body)/1_000_000)s")
```

Comment:
We should probably do floating point arithmetic here

Reply:
Indeed, `TimeInterval` does.